testsuite/simulation/modelica/spatialDistribution/bigSteps.mos

```diff
@@ -0,0 +1,55 @@
+// name:                bigSteps.mos
+// keywords:            spatialDistribution
+// status:              correct
+// teardown_command:    rm -f bigStepsModel*
+//
+// Simple modle with fast velocity and big steps.
+
+loadModel(Modelica); getErrorString();
```

Comment:
maybe load a specific version?

Reply:
The newest version of MSL should be fine. If it breaks at some point we need to fix that.
Not sure if it is a good idea to test for older MSL versions as well.

Reply:
Ok. Just making sure.